# Custom Message to store the IMU data
uint64 timestamp # Timestamp of the message
float64 imu1     # IMU data from arm 1
float64 imu2     # IMU data from arm 2
float64 imu3     # IMU data from arm 3
float64 imu4     # IMU data from arm 4
